pkg/release/release.go

```diff
@@ -287,6 +312,26 @@ next:
 
 		action = TestAction
 		goto next
+	case Migrate2to3Action:
+		logger.Log("info", "running 2to3 migration", "phase", action)
+
+		converter := v3.Converter{
+			// TODO: pass the tiller namespace to converter
+			TillerNamespace: "",
+		}
+		err := converter.Convert(hr.GetReleaseName())
+
+		if err != nil {
+			status.SetStatusPhase(r.hrClient.HelmReleases(hr.Namespace), hr, v1.HelmReleasePhaseFailed)
+			err = fmt.Errorf("failed to convert helm chart from v2 to v3: %w", err)
+			logger.Log("error", err, "phase", action)
+			errs = append(errs, err)
+			break
+		}
```

Comment:
Can you restructure this to a `switch` statement? This would allow new version converters to be added (i.e. a future v3 -> v4).